### ISSUE TITLE
Feature Request: Optional Auto Login with saved credentials

### ISSUE THREAD

**Issue by LoVega1337** [leonardoxh/race-control-tv#137]
Since F1TV is automatically logging out sessions after ~5 days, it would make sense to implement an optional Auto Login feature with given Credentials by the user. 
Basically the user should provide the credentials in the Settings, enable the Auto Login feature and if the token expires and the WebView opens on app start, the WebView should auto fill in given Credentials on the F1TV page and log in. It should only be tried once in case the log in fails e.g. wrong credentials, etc.
I have looked into it and it is actually possible with some JavaScript.

**Comment by leonardoxh** [comment]
That is certainly a very nice feature

**Comment by satbeginner** [comment]
Hi, I support this request strongly. 

We use the app on our TCL Android TV and it works superduper great. 👍👍👍

But I have to say, loging in with the remote having a long email address and a long, but safe password is a bit fiddly.

Thanks for this great app!!

Leo

**Comment by jeroenvandeleur** [comment]
Yes, this would be awesome! I also have a long username and password

**Comment by cmbsolutions** [comment]
I would like this to be added also. Entering a strong password with a sluggish remote every weekend Just s...s
